### PR TITLE
Added Play Button on every flip card present on the webpages.

### LINKED ISSUE #10: Add "play now" button when movie card flips

### PR DESCRIPTION
**Pull request by rompyroshni31**
#### Changes I've made
Added play button on each and every flip card present on the website. Adjusted the elements according to every card.

#### Changes on my GitHub page:
Check the deployment link here.
https://rompyroshni31.github.io/Movie-Streaming-Website/

#### Any unusual behavior of your code:
NA

DCP 2021
DEVINCEPT 2021 2

#### Checklist

- [x] I have mentioned related issue numbers in the PR description.
- [x] I have added screenshots and related demo links of my changes.
- [x] The title of my pull request is a short description of the requested changes.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] DCP 2021

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
Check one peice movie poster in Anime section in home page

Check fury poster on war page.

**Comment by rompyroshni31** [comment]
Done!

**Comment by QAZIMAAZARSHAD** [comment]
image

image

**Comment by QAZIMAAZARSHAD** [comment]
Correct these 2

**Comment by rompyroshni31** [comment]
Done.

**Comment by QAZIMAAZARSHAD** [comment]
image


Not reflected in your link.

File: home.html [hunk 1]
```diff
@@ -2505,7 +2572,7 @@ <h2 class="footer-heading">Services</h2>
         <div class="footer-middle">
           <h2 class="footer-heading">Try Our App</h2>
           <ul class="footer-middle-list icons">
-            <a class="footer-links" href="#"><i class="fab fa-google-play" style="color:red"></i></a>
+            <a class="footer-links" href="#"><i class="fab fa-google-Play" style="color:red"></i></a>
```
Comment (QAZIMAAZARSHAD) [inline home.html]: Correct it.

**Comment by rompyroshni31** [comment]
Done.

**Comment by QAZIMAAZARSHAD** [comment]
@rompyroshni31 you have not filled the google form.

Without it, your PR will not be counted.

Fill it as soon as possible. If you are a part of DCP.

**Comment by QAZIMAAZARSHAD** [comment]
Once you fill it ping me here.

**Comment by rompyroshni31** [comment]
Done!

**Pull request merged**